parm/atm/hofx/hofx4d.yaml.j2

```diff
@@ -37,8 +37,12 @@ model:
   filename_sfcd: '%yyyy%mm%dd.%hh%MM%ss.sfc_data.nc'
   filename_sfcw: '%yyyy%mm%dd.%hh%MM%ss.fv_srf_wnd.res.nc'
   filename_cplr: '%yyyy%mm%dd.%hh%MM%ss.coupler.res'
-  tstep: $(BKG_TSTEP)
+  tstep: '{{ BKG_TSTEP }}'
```

Comment:
If this is a number, it should not be quoted.

Reply:
Thanks for checking @RussTreadon-NOAA.  Even though this is a string, we don't necessarily need to retain the quotes.  I am thinking we could/should cast this type of strings into a `datetime.timedelta`.

Reply:
Quotes removed at [aac4d0d](https://github.com/NOAA-EMC/GDASApp/commit/aac4d0da64d18c0874ea2b75d1662d91bdd1db31).  I don't know if anyone uses this yaml any more.  I could not find any reference to this yaml in `test/`, `ush/`, or `utils/`.